src/libmeasurement_kit/ooni/ooni_test.cpp

```diff
@@ -69,6 +69,7 @@ void OoniTest::geoip_lookup(Callback<> cb) {
     probe_cc = "ZZ";
     ip_lookup(
         [=](Error err, std::string ip) {
+            // TODO: refactor to use the if-err-then-cb-and-return style
```

Comment:
Yes, indeed, that's why I'd like to refactor! I've created #815 to take care of this.